rules/carbonblack_rules/cb_audit_api_key_created_retrieved.py

```diff
@@ -0,0 +1,21 @@
+PATTERNS = (
+    " retrieved secret for API ID ",
```

Comment:
Are the spaces here intentional?

Reply:
Yes, probably not strictly necessary but it is how it appears in the logs